remoteappmanager/templates/page.html

```diff
@@ -52,79 +50,80 @@
-                    {% block login_widget %}
-                    <a id="logout" href="{{ logout_url }}" class="btn btn-default btn-flat"><i class="fa fa-sign-out"> Sign out</i></a>
-                    {% endblock %}
-                  </div>
-                </li>
-              </ul>
-            </li>
-          </ul>
-        </div>
+        {% if user is not None %}
+          <!-- Navbar Right Menu -->
+          <div class="navbar-custom-menu">
+            <ul class="nav navbar-nav">
+              <!-- User Account Menu -->
+              <li id="user-menu" class="dropdown user user-menu">
+                <!-- Menu Toggle Button -->
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                  <!-- The user image in the navbar-->
+                  <img src="https://www.gravatar.com/avatar/{{ gravatar_id }}" class="user-image" alt="User Image">
+                  <!-- hidden-xs hides the username on small devices so only the image appears. -->
+                  <span class="hidden-xs">{{ user.name }}</span>
+                </a>
+                <ul class="dropdown-menu">
+                  <!-- The user image in the menu -->
+                  <li class="user-header">
+                    <img src="https://www.gravatar.com/avatar/{{ gravatar_id }}" class="img-circle" alt="User Image">
+                    <p>
+                      {{ user.name }}
+                    </p>
+                  </li>
+                  <!-- Menu Body -->
+                  <li class="user-body">
+                  </li>
+                  <!-- Menu Footer-->
+                  <li class="user-footer">
+                    <div class="pull-right">
+                      {% block login_widget %}
+                      <a id="logout" href="{{ logout_url }}" class="btn btn-default btn-flat"><i class="fa fa-sign-out"> Sign out</i></a>
+                      {% end %}
+                    </div>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        {% end %}
       </nav>
     </header>
 
     <aside class="main-sidebar">
       {% block sidebar %}
-      {% endblock %}
+      {% end %}
     </aside>
 
     <div class="content-wrapper">
       <!-- Main content -->
       {% block main %}
-      {% endblock %}
+      {% end %}
     </div>
   </div>
 
   <div id="error-dialog-container"></div>
 
   <script src="{{ static_url("dist/sharedDependencies.js") }}"></script>
 
+  <script type="text/javascript">
+    window.apidata = {
+      base_url: "{{ base_url }}",
+      {% if user is not None %}
+        user: "{{ user.name }}",
+      {% end %}
+      {% if analytics is not None %}
+        analytics: {
+          tracking_id: "{{ analytics['tracking_id'] }}"
+        },
+      {% end %}
+    }
+  </script>
   {% block script_init %}
-    <script type="text/javascript">
-      window.apidata = {
-        base_url: "{{base_url}}",
-        prefix: "{{prefix}}",
```

Comment:
Jinja wasn't complaining but tornado is less permissive. With tornado code in the brackets is python code so it raises a `name "prefix" is not defined`